gapic-generator-java-pom-parent/pom.xml

```diff
@@ -27,7 +27,7 @@
         consistent across modules in this repository -->
     <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
     <grpc.version>1.68.1</grpc.version>
-    <google.auth.version>1.29.0</google.auth.version>
+    <google.auth.version>1.30.0</google.auth.version>
```

Comment:
This can be reverted now since https://github.com/googleapis/sdk-platform-java/pull/3367 is in.